src/lib.rs

```diff
@@ -31,6 +31,7 @@ pub enum Error {
     Http(http::Error),
     Reqwest(reqwest::Error),
     Url(url::ParseError),
+    Serde(serde_json::error::Error)
```

Comment:
I don't think you need the serde error here. The `reqwest::Error` wraps the serde `Error`

Reply:
NVM this is for the `/units-for-slot` and not the `MarketApi`, my bad!